src/common/components/Modal/ModalContent/ModalContent.tsx

```diff
@@ -0,0 +1,68 @@
+import { Icon } from "@iconify-icon/react";
```

Comment:
thought:
somehow, icons generated from `@iconify-icon/react` would blink on refresh. shall we stick with copying the required svgs and use it as `CustomIcons`?

![screenshot_20240120_184657](https://github.com/AfterClass-io/afterclass.io-v2/assets/13061926/3d886bb5-3fda-44f8-8804-74787482b6cc)